src/intents.js

```diff
@@ -46,6 +46,14 @@ function injectService (url, element, intent, data) {
         return event.source.postMessage(data, event.origin)
       }
 
+      if (handshaken && event.data.type === `intent-${intent._id}:size`) {
+        ['width', 'height', 'maxWidth', 'maxHeight', 'minWidth', 'minHeight'].forEach(prop => {
```

Comment:
After thinking @doubleface, I can't see any cases when we will need max and min css properties here. I was first thinking about responsive behaviour but, even that, if we use the service from cozy-collect, the app which handle the inter-app view will the provide a responsive width and height. So I think we would just need width and height here. What do you think? And @y-lohse ?